app/models/metric/targets.rb

```diff
@@ -70,14 +70,10 @@ def self.capture_storage_targets(hosts)
     hosts.flat_map(&:storages).uniq.select { |s| Storage.supports?(s.store_type) & s.perf_capture_enabled? }
   end
 
-  def self.capture_vm_targets(targets, parent_class, options)
-    enabled_parents = targets.select do |t|
-      t.kind_of?(parent_class) &&
-        t.kind_of?(Metric::CiMixin) &&
-        t.perf_capture_enabled? &&
-        t.respond_to?(:vms)
```

Comment:
@Fryguy 100% agreed. But this is the logic.
And if I change it, tests fail.

Part of the reason why I'm making the code look this way is to show the contradictions